pkg/hs/urbit-king/lib/Urbit/Vere/Dawn.hs

```diff
@@ -292,54 +269,32 @@ removePrefix withOhEx
  where
   (prefix, suffix) = splitAt 2 withOhEx
 
-chunkBytestring :: Int -> ByteString -> [ByteString]
-chunkBytestring size bs
-  | null rest = [cur]
-  | otherwise = (cur : chunkBytestring size rest)
- where
-  (cur, rest) = splitAt size bs
-
-data TurfRequest = TurfRequest
-  { trqHexBlockNum :: TextBlockNum
-  , trqTurfId :: Int
-  } deriving (Show, Eq)
+data TurfRequest = TurfRequest ()
```

Comment:
Why is there a `()` here? This isn't OCaml.

Reply:
Is it needed for a from/toNoun to get the right representation? That would make it legit. Otherwise I think not needed?